flax/experimental/nnx/nnx/transforms.py

```diff
@@ -152,7 +157,9 @@ class JITOptions:
   backend: tp.Optional[str]
   inline: bool
   abstracted_axes: tp.Optional[tp.Any]
+  # nnx specific
   donate_object_state: bool
+  constrain_object_state: tp.Callable[[State], State] | None
```

Comment:
Please document what this field does either here or elsewhere.

Reply:
Done